testhelpers/image_privileges.go

```diff
@@ -25,16 +26,19 @@ const (
 // NewImagePrivileges(Readable) returns ImagePrivileges{readable: true, writable: false}
 // NewImagePrivileges(Writable) returns ImagePrivileges{readable: false, writable: true}
 // NewImagePrivileges(Readable, Writable) returns ImagePrivileges{readable: true, writable: true}
-func NewImagePrivileges(imageAccess ...ImageAccess) (priv ImagePrivileges) {
+func NewImagePrivileges(imageAccess ...ImageAccess) (ImagePrivileges) {
```

Comment:
You don't need the `()` around the return type

Reply:
Removed